spec/services/quotes_service_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe QuotesService do
+  xit 'can return a list of quotes with a given keyword' do
```

Comment:
I'm curious why you skipped this test? When I make an API call with the query 'test' it returns 6 results (not 10) but that is also what the failure error says when I run your test suite. So you could just update line 10 so the expected count is 6. 

Alternatively you could go back to the query phrase "I am" that is used in the initial feature spec where you already know 10 results will be returned. Either of those changes would allow this test to pass, unless I'm missing some other reason why you skipped it.